extensions/default/src/Panels/createReportDialogPrompt.tsx

```diff
@@ -2,11 +2,16 @@ import PROMPT_RESPONSES from '../utils/_shared/PROMPT_RESPONSES';
 
 export default function CreateReportDialogPrompt({
   title = 'Create Report',
+  modality = 'SR',
+  minSeriesNumber,
```

Comment:
please add documentation above this function , and also migration guide about what is changed here (which is if not defined series number is picked as 4000)